### ISSUE TITLE
new generator (contact here?)

### ISSUE THREAD

**Issue by Killedragon** [kbuzsaki/bingosync#165]
all right so
i have a list for the idea of a new game bingo generator for bingosync, and the about page says that i could send it to him, saltor, and about contacting him here (i cant message him on discord)
do i send here the pastebin with the list and name for the name or kind of?
thanks

**Comment by kbuzsaki** [comment]
Yes, send the goal list and game details here and I can add them

**Comment by Killedragon** [comment]
okey million thanks!
"Doodle Champion Island Games Begin!"
https://pastebin.com/p0v6VkfG
the full game for the google browser that google created for the olympic games
in the speedrunning server saw some conversation happend about bingo stuff and had the idea for this page
